Source/WebCore/page/ElementTargeting.cpp

```diff
@@ -260,70 +262,76 @@ Vector<TargetedElementInfo> findTargetedElements(Page& page, TargetedElementRequ
         onlyMainElement = &descendant;
     }
 
+    auto isCandidate = [&](Element& element) {
+        return element.renderer()
```

Comment:
since this is in a lambda anyways, it could be a bit more readable if this compound && could be broken up into multiple early returns maybe?

Reply:
Sounds good to me!